tests/integration/test_delete_simulation.py

```diff
@@ -0,0 +1,83 @@
+import pytest # skipcq: PY-W2000
+from sqlalchemy.orm.scoping import scoped_session
+from yaptide.utils.enums import EntityState, InputType, SimulationType
+from yaptide.persistence.models import (
+    YaptideUserModel,
+    CelerySimulationModel,
+)
+import json
+
+
+@pytest.fixture(autouse=True)
```

Comment:
We don't have so many of `autouse` fixtures. To be more consistent with other pieces of the code, define a fixture with function scope and require is explicitely as a test method argument,